_includes/head.html

```diff
@@ -65,7 +65,6 @@
     {% else %}
     <meta property="twitter:image" content="https://expressjs.com/images/og.png">
     {% endif %}
-    <script data-cfasync="false" src="//ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
     <script data-cfasync="false" src="/js/app.js"></script>
```

Comment:
```suggestion
    <script data-cfasync="false" defer src="/js/app.js"></script>
```

We can use defer here, and we won’t need to use `document.addEventListener("DOMContentLoaded", function () {` anymore (https://github.com/expressjs/expressjs.com/blob/c327d43be63208fbcabf8dc7aca15a2394de1b51/js/app.js#L1)

Reply:
Yes, we should use `defer`. But doesn't `data-cfasync="false"` do the same ? Optimize page load performance.

Reply:
This is worth discussing with more people, because up to now we've always disabled [Rocket Loader](https://developers.cloudflare.com/speed/optimization/content/rocket-loader/) using that property (data-cfasync="false"). In the meantime, it would be better to use defer. I've even considered proposing that we disable[ Rocket Loader](https://developers.cloudflare.com/speed/optimization/content/rocket-loader/) entirely, but I haven't done so yet since I haven't fully analyzed how it works.

Reply:
Let's do it in separate PR. Since doesn't make sense to use defer on single file. I am not going to change all other files.

Reply:
I don’t understand, we’re already using defer on the other JavaScript files we import, and that’s how it’s being handled. We’ve manually disabled Rocket Loader because it previously caused issues (see https://github.com/expressjs/expressjs.com/issues/1508), so using defer here makes complete sense and should be part of this PR.

whether or not to use Rocket Loader is a separate discussion, but I’d really opt for keeping it disabled.

Reply:
Oh, I noticed only top theme.js file. menu and copy code are deferred. 😂 Funny is copy code written by me, I don't remember. 🤣🤣

Reply:
Let's add 'defer' 👍